### PR TITLE
Add Chinese translation

### PR DESCRIPTION
**Pull request by HanJiang-cn**

### CODE REVIEW

File: src/main/resources/assets/energizedpower/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+
+  "book.energizedpower.page.sawdust.1": "锯末是锯木机产出的副产品，锯末可以来制作锯末块。",
+  "book.energizedpower.page.sawdust.2": "锯末块可以烧制成木炭。",
+
+  "book.energizedpower.page.energized_copper_ingot.1": "充能铜锭是充能站这样高级的机器需要的。",
+  "book.energizedpower.page.energized_copper_ingot.2": "充能站是用这个锭做的，那倒不是吗？",
+  "book.energizedpower.page.energized_copper_ingot.3": "充电站可以用来制作这种锭的。",
+  "book.energizedpower.page.energized_copper_ingot.4": " [但它很昂贵]",
+
+  "book.energizedpower.page.energized_gold_ingot.1": "充能金锭应用于高级的机器（e.g.充电站）。",
```
Comment (JDDev0) [inline src/main/resources/assets/energizedpower/lang/zh_cn.json:347]: Translation for e.g. (meaning `for example`) [Optional]

**Review by JDDev0 — CHANGES_REQUESTED** [review]

**Comment by JDDev0** [comment]
Thanks for your contribution!

The changes will be published in v1.4.5!
But before I can merge this update, please finish the tasks I have written in the review (At least those without [Optional]).
Thanks!

**Review by HanJiang-cn — COMMENTED** [review]
I am very sorry, but due to my limited abilities, this section was a little difficult to translate and I will keep the original text as per the rules of our translation mod. However, in the future I will ask other people to assist me in translating this section.
I'll fix the other issues, thanks for pointing them out!

**Comment by JDDev0** [comment]
No problem!

**Review by JDDev0 — APPROVED** [review]

**Pull request merged**